pilot/pkg/networking/core/v1alpha3/configgen.go

```diff
@@ -30,6 +31,9 @@ type ConfigGeneratorImpl struct {
 	// Must be rebuilt for each push epoch
 	PrecomputedOutboundClusters map[string][]*xdsapi.Cluster
 	// TODO: add others in future
+
+	// MixerPerRouteFilterConfig has mixer Router Filter config, indexed by service hostname + outbound/inbound
+	MixerPerRouteFilterConfig map[string]*types.Struct
```

Comment:
Just call it PerRouteFilterConfig map[pluginName]interface{}
This will let other plugins cache their data as well

Reply:
Done~